### ISSUE TITLE
Contribute MessagePackSerializer upstream

### ISSUE THREAD

**Issue by dan-blanchard** [Yelp/pyleus#159]
Your [MessagePackSerializer](https://github.com/Yelp/pyleus/blob/develop/topology_builder/src/main/java/com/yelp/pyleus/serializer/MessagePackSerializer.java) class seems like it would be broadly useful in the Storm community.  Would you be willing to contribute it to Apache Storm proper?  It's already Apache license, and obviously so is their code.

**Comment by dan-blanchard** [comment]
I don't mean to step on any toes here, but since this is so broadly useful and the licenses are compatible, I was planning on trying to get this file added upstream.  I would prefer to have the :ok_hand: from you all first, but in the end it's all just compatible open source code.

**Comment by patricklucas** [comment]
Yes, of course! Sincere apologies for missing this question from October, thanks to @poros for pinging me about it.

I'm totally on board with sending this upstream, though I'd like to get +1s from @ecanzonieri and @poros as well.

**Comment by dan-blanchard** [comment]
Coincidentally, it looks like someone has a slightly different serializer implementation they just opened a PR for, so I'm sure your input on their PR would be very valuable.  It's apache/storm#1136.
